releases/release-8.1.0.md

```diff
@@ -0,0 +1,358 @@
+
+    In v8.0.0, TiDB Lightning deprecates the [old version of conflict detection](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#the-old-version-of-conflict-detection-deprecated-in-v800) strategy for the physical import mode, enables you to control the conflict detection strategy for both logical and physical import modes via the [`conflict.strategy`](/tidb-lightning/tidb-lightning-configuration.md) parameter (experimental), and simplifies the configuration of this parameter. In addition, in the physical import mode, the `replace` strategy supports retaining the latest data and overwriting the old data when the import detects data with primary key or unique key conflicts. In v8.1.0, the capability to handle conflicting data with the `replace` strategy becomes generally available (GA).
+
+    For more information, see [documentation](/tidb-lightning/tidb-lightning-configuration.md).
+
+* TiCDC supports client authentication [#10636](https://github.com/pingcap/tiflow/issues/10636) @[CharlesCheung96](https://github.com/CharlesCheung96)
+
+    In v8.1.0, TiCDC supports client authentication when you are using the TiCDC CLI or OpenAPI. This feature enables you to configure TiCDC to require client authentication using client certificates, thereby establishing mutual Transport Layer Security (mTLS). Additionally, you can configure authentication based on TiDB username and password.
+
+    For more information, see [documentation](/ticdc/ticdc-client-authentication.md).
+
+## Compatibility changes
+
+> **Note:**
+>
+> This section provides compatibility changes you need to know when you upgrade from v8.0.0 to the current version (v8.1.0). If you are upgrading from v7.6.0 or earlier versions to the current version, you might also need to check the compatibility changes introduced in intermediate versions.
+
+### Behavior changes
+
+* In earlier versions, the `tidb.tls` configuration item in TiDB Lightning treats values `"false"` and `""` the same, as well as treating the values `"preferred"` and `"skip-verify"` the same. Starting from v8.1.0, TiDB Lightning distinguishes the behavior of `"false"`, `""`, `"skip-verify"`, and `"preferred"` for `tidb.tls`. For more information, see [TiDB Lightning configuration](/tidb-lightning/tidb-lightning-configuration.md).
+* In earlier versions, when tidb exit, the primary node of autoid service would execute `forceRebase` for tables with AUTO_ID_CACHE=1 attribute, the purpose of this behaviour is to ensure the allocated autoid been continuous. However, when there are a lot of tables with AUTO_ID_CACHE=1 attribute, the `forceRebase` operation takes a long time, blocking the restart of that tidb. In the current version, the `forceRebase` behavior is removed, so it may cause 'hole' in the allocated autoid, but tidb would not block when restart.
```

Comment:
```suggestion
* For tables configured with `AUTO_ID_CACHE=1`, TiDB supports [centralized allocation of auto-increment IDs](/auto-increment.md#mysql-compatible-mode). In earlier versions, the "primary" TiDB node of this service would automatically perform a `forceRebase` operation upon TiDB process exit (for example, when the TiDB node restarts) to ensure that auto-assigned IDs are as consecutive as possible. However, when there are too many tables configured with `AUTO_ID_CACHE=1`, executing `forceRebase` becomes very time-consuming, causing TiDB to be unable to restart promptly after receiving the exit signal, and even blocking data writes, thus affecting system availability. Therefore, starting from v8.1.0, TiDB has removed the `forceRebase` behavior, resolving the issue of the process being difficult to exit when restarting the TiDB node in this scenario, but it will result in a small number of auto-assigned IDs being non-consecutive during primary-secondary switchovers.
```